pages.id/common/curl.md

```diff
@@ -0,0 +1,37 @@
+# curl
+
+> Mentransfer data dari atau ke server.
+> Mendukung sebagian besar protokol, termasuk HTTP, FTP, dan POP3.
+> Informasi selengkapnya: <https://curl.haxx.se>.
+
+- Unduh konten URL ke file:
+
+`curl {{http://example.com}} -o {{filename}}`
```

Comment:
Could you translate the arguments like `filename` to be easier for non-English speakers?